src/main/java/com/epam/poc/commons/BasePage.java

```diff
@@ -192,20 +191,33 @@ public void clickToElementByJS(WebDriver driver, WebElement element) {
 
     public void clickToElementByJS(WebDriver driver, By by) {
         JavascriptExecutor executor = (JavascriptExecutor) driver;
-        executor.executeScript("arguments[0].click();", getElement(driver,by));
+        executor.executeScript("arguments[0].click();", getElement(driver, by));
     }
 
-    public void navigateToPreviousPage(WebDriver driver){
+    public void navigateToPreviousPage(WebDriver driver) {
         driver.navigate().back();
     }
 
     public void waitForPageTitleAsExpectedText(WebDriver driver, String pageTitleValue) {
         try {
             WebDriverWait wait = new WebDriverWait(driver, Long.parseLong(propertyReader.getValue(GlobalConstants.LONG_TIMEOUT_KEY)));
             wait.until(ExpectedConditions.titleIs(pageTitleValue));
-        }
-        catch (Exception e) {
+        } catch (Exception e) {
             logger.info("Page title is not the same as expected text");
         }
     }
+
+    public void switchToNewTab(WebDriver driver) {
+        ArrayList<String> tabs = new ArrayList(driver.getWindowHandles());
+        driver.switchTo().window(tabs.get(tabs.size() - 1));
+    }
+
+    public void switchToPreviousTab(WebDriver driver) {
```

Comment:
Hi Nhi, you should change the name of method to "switchToFirstTab" because of this command "driver.switchTo().window(tabs.get(0))"